app/views/user_mailer/new_user_in_leader_area.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= @new_user.name %>, has joined. They're located within 50 miles of your chapter! Please email them at
+<%= @user_user.email %> and welcome them, and consider providing info on the next local meetup.
+Let's encourage in-person involvement within Operation Code!
```

Comment:
Nice.  Did you confirm this email's subject and body text with @hollomancer @kylemh and/or @jenn?

Reply:
Yes me and @kylemh talked about the body last night. I created the subject.